src/lib.rs

```diff
@@ -81,6 +81,65 @@ impl Compiler {
         })
     }
 
+    fn get_orig_src_map(
+        &self,
+        name: &FileName,
+        input_src_map: &InputSourceMap,
+    ) -> Result<Option<sourcemap::SourceMap>, Error> {
+        self.run(|| -> Result<_, Error> {
+            // Load original source map
+            match input_src_map {
+                InputSourceMap::Bool(false) => Ok(None),
+                InputSourceMap::Bool(true) => {
+                    // Load original source map if possible
+                    match &name {
+                        FileName::Real(filename) => {
+                            let path = format!("{}.map", filename.display());
+                            let file = File::open(&path)
+                                .context("failed to open input source map file")?;
+                            Ok(Some(sourcemap::SourceMap::from_reader(file).with_context(
+                                || format!("failed to read input source map from file at {}", path),
+                            )?))
+                        }
+                        _ => {
+                            log::error!("Failed to load source map for non-file input");
+                            return Ok(None);
+                        }
+                    }
+                }
+                InputSourceMap::Str(ref s) => {
+                    if s == "inline" {
+                        // Load inline source map by simple string
+                        // operations
+                        let s = "sourceMappingURL=data:application/json;base64,";
+                        let idx = s.rfind(s);
```

Comment:
This doesn't look right. Typo?

Reply:
@95th Oh... I'll fix it soon. Thank you very much.